bin/internal/shared.bat

```diff
@@ -132,6 +132,8 @@ GOTO :after_subroutine
     REM Do not fall through - return from subroutine
     EXIT /B
 
+  SET retries="0"
```

Comment:
This can be removed.  The script never flows through here - it enters `do_sdk_update_and_snapshot` via a GOTO.

The `SET /A retries+=1` will set the variable to 1 if it does not already have a value.